docs/src/design/stack/crypto_ops.md

```diff
@@ -178,4 +178,28 @@ After calling the `mem_stream ` with `x_ptr`, the operation does the following:
 > TODO: add detailed constraint descriptions. See discussion [here](https://github.com/0xPolygonMiden/miden-vm/issues/869).
 
 The effect on the rest of the stack is:
-* **No change.**
\ No newline at end of file
+* **No change.**
+
+The `RCOMBBASE` makes two memory access request. To simplify the description of these, we first define the following variables :
+
+$$
+v_1 = \sum_{i=0}^3\alpha_{i+5} \cdot h_{i}'
+$$
+
+$$
+v_2 = \sum_{i=0}^1\alpha_{i+5} \cdot h_{i + 4}'
+$$
```

Comment:
Are $h_i'$ and $h_{i+4}'$ correct here? I thought helper values come from the current row?

Also, for $v_2$ I think we are missing $\alpha_7$ and $\alpha_8$ terms.

Reply:
Yes, that's a typo. Fixed now!
For the remaining alpha terms, I have omitted those as they are multiplied by `ZERO`, should I include them and add a note stating the aforementioned fact?